ui/package.json

```diff
@@ -32,11 +32,12 @@
     "format:types": "prettier --write \"types/**/*.{d.ts,ts,json}\"",
     "test:create": "node ./test/cypress/helpers/create-spec.js -c",
     "test:build": "node build/script.build.js js api && node build/script.build.js js transforms",
-    "test:e2e": "yarn test:build && cd ./dev && cross-env NODE_ENV=test start-test \"node ./script.dev.js\" http-get://localhost:9000 \"cypress open --e2e\" && cd ..",
-    "test:e2e:ci": "yarn test:build && cd ./dev && cross-env NODE_ENV=test start-test \"node ./script.dev.js\" http-get://localhost:9000 \"cypress run --e2e\" && cd ..",
-    "test:component": "yarn test:build && cd ./dev && cross-env NODE_ENV=test cypress open --component && cd ..",
-    "test:component:run": "yarn test:build && cd ./dev && cross-env NODE_ENV=test cypress run --component && cd ..",
-    "test:component:ci": "yarn test:build && cd ./dev && cross-env NODE_ENV=test cypress run --component --record --parallel && cd .."
+    "test:e2e": "yarn test:build && cross-env NODE_ENV=test start-test \"node ./dev/script.dev.js\" http-get://localhost:9000 \"cypress open --project ./dev --e2e\"",
+    "test:e2e:run": "cross-env NODE_ENV=test start-test \"node ./dev/script.dev.js\" http-get://localhost:9000 \"cypress run --project ./dev --e2e\"",
+    "test:e2e:ci": "yarn test:build && yarn test:e2e:run",
+    "test:component": "yarn test:build && cross-env NODE_ENV=test cypress open --project ./dev --component",
+    "test:component:run": "cross-env NODE_ENV=test cypress run --project ./dev --component",
```

Comment:
Previously `ci` was useless locally, and `run` was used to achieve the same locally. Now, we need a script that doesn't use the build script to avoid having to re-build the package when running tests, primarily to be used in CI, but also has a use locally. You don't have to rebuild the package if you only change the tests. I also updated the documentation to explain the updated purposes of the scripts.

I think of the `:ci` scripts as a way of mimicking the CI-way, usable locally and in CI. If the CI has very CI-specific settings, it can set env variables, pass extra arguments to the script, or override it without changing it completely(_when the first two ways are impossible_).

Reply:
> You don't have to rebuild the package if you only change the tests.

You usually run tests even when working on the code and sometimes update the code when working on tests.
We automated the build step to avoid people forgetting the build step and ending up catching it in the CI instead, since it happened many times to me and others who contributed while working on the feature

I'll update the docs to mention `ci` version as the one to use, and `run` as a more granular and advanced one for special cases